OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -195,90 +214,49 @@
 <hr/>
 
 <h3 class="wiki-nav-item pb-2" id="CTL-x100-android-mode">
-    CTL-x100 is not detected
+    My CTL-x100 is not detected?
 </h3>
 <div class="ms-3">
 <p>
-    CTL-x100 tablets boots in Android mode instead of PC mode in some rare cases. To fix this, press
+    It is possible for CTL-x100 tablets boots in Android mode instead of PC mode in some rare cases. To fix this, press
     the outer 2 express keys for 3-4 seconds until the LEDs change brightness. This toggles the tablet's operating mode
     between PC (high LED brightness) and Android mode (low LED brightness).
+    <small class="ms-3 text-muted">
+        Note: If you are sure your tablet is in PC mode, please follow our generalised instructions <a href="#tablet-not-detected">here</a>.
+    </small>
+
 </p>
 
 </div>
 
 <hr/>
 
-<h3 class="wiki-nav-item pb-2" id="display-session-not-detected">
-    X11/Wayland Session not detected
+<h3 class="wiki-nav-item pb-2" id="systemd-autostart">
+    The systemd user service doesn't automatically start?
 </h3>
 <div class="ms-3">
-    <h4>X11 <small class="text-muted">(systemd)</small></h4>
-    <div class="mx-3">
-        <p>
-            Your <code>.xinitrc</code> is likely configured improperly and the <code>DISPLAY</code> environment
-            variable isn't set for systemd.<br/>
-            Proper configuration for the <code>.xinitrc</code> file can be found
-            <a href="https://wiki.archlinux.org/index.php/Xinit#Configuration">here</a>.<br/>
-            <small class="ms-3 text-muted">
-                Note: The fix below is just a quick and dirty fix to get OpenTabletDriver running and should
-                instead be resolved as above.
-            </small>
-        </p>
-        <ol>
-            <li>
-                Create an override file
-                <codeblock class="mt-2" language="bash">
-                    # Opens with whatever program is set to $EDITOR
-                    systemctl --user edit opentabletdriver.service
-                </codeblock>
-            </li>
-            <li>
-                Add this block below the first 2 lines of the file, then save and exit
-                <codeblock class="mt-2" language="ini">
-                    [Service]
-                    Environment=DISPLAY=:0
-                </codeblock>
-            </li>
-            <li>
-                Finally, run the following commands
-                <codeblock class="mt-2" language="bash">
-                    # Reload the systemd user daemon
-                    systemctl --user daemon-reload
-                    # Start the OpenTabletDriver daemon service (restarts if already running)
-                    systemctl --user restart opentabletdriver.service
-                </codeblock>
-            </li>
-        </ol>
-    </div>
+    <p>
+        If you have enabled the systemd user service and this happens, your window manager isn't properly setting up
```

Comment:
```suggestion
        If you have enabled the systemd user service and this happens, your Linux distribution isn't properly setting up
```

technically not a wm responsibility and on Arch I believe it is expected of the user to fix this themselves, since it does not seem like even the mainstream DE's target it properly on Arch